sp800-63b/sec4_aal.md

```diff
@@ -82,6 +82,8 @@ Communication between the claimant and verifier (the primary channel in the case
 
 Verifiers operated by government agencies at AAL2 SHALL be validated to meet the requirements of [FIPS 140](#FIPS140-2) Level 1.
 
+When a device such a smartphone is used in the authentication process, the unlocking of that device (typically done using a PIN or biometric) SHALL NOT be considered to satisfy one of the authentication factors. This is because it is generally not possible for verifier to know that the device had been locked, nor whether the unlock process met the requirements for the relevant authenticator type.
```

Comment:
Normative language kills us here grammatically.  2 ideas:

SHALL NOT satisfy one of the factors.  Or:

Authentication factors SHALL be used only after the mobile device is is unlocked.

In addition, we should consider broadening.  Even a workstation login is considered an unlocking.